File: HomeBuildApp/workflows/src/main/java/com/r3/developers/csdetemplate/homebuild/workflows/CreateRequestWorkStateArgs.java
package com.r3.developers.csdetemplate.homebuild.workflows;

public class CreateRequestWorkStateArgs {
    private double budget;
    private int age;
    private int houseHold;
    private String address;

    private String consultant;


    private String municipality;


    public CreateRequestWorkStateArgs() {
    }

    public CreateRequestWorkStateArgs(double budget, Integer age, Integer houseHold, String address, String consultant, String municipality) {
        this.budget = budget;
        this.age = age;
        this.houseHold = houseHold;
        this.address = address;
        this.consultant = consultant;
        this.municipality = municipality;
    }

    public double getBudget() {
        return budget;
    }


    public int getAge() {
        return age;
    }

    public int getHouseHold() {
        return houseHold;
    }

    public String getAddress() {
        return address;
    }

    public String getConsultant() {
        return consultant;
    }

    public String getMunicipality() {
        return municipality;
    }
}

File: HomebuildKotlin/workflows/src/main/kotlin/com/r3/developers/csdetemplate/homebuild_Workflows/CreateRequestWorkStateArgs.kt
package com.r3.developers.csdetemplate.homebuild_Workflows
class CreateRequestWorkStateArgs {
    var budget = 0.0
        private set
    var age = 0
        private set
    var houseHold = 0
        private set
    var address: String? = null
        private set
    var consultant: String? = null
        private set
    var municipality: String? = null
        private set

    constructor()
    constructor(
        budget: Double,
        age: Int,
        houseHold: Int,
        address: String?,
        consultant: String?,
        municipality: String?
    ) {
        this.budget = budget
        this.age = age
        this.houseHold = houseHold
        this.address = address
        this.consultant = consultant
        this.municipality = municipality
    }
}